Tensile/Tests/extended/direct_to_lds/dtl_tsgr_f8.yaml

```diff
@@ -1,5 +1,5 @@
 TestParameters:
```

Comment:
This test is specific to gfx94x. If we skip this, this file is useless.

Reply:
Yea, that was intentional. Currently, new f8 work is happening in TensileLite repo, and there are still a few tests for f8 features in other files in extended tests. Maybe I should have moved the file to disabled rather than mark as skip, but goal was to reduce f8 tests being run. I can re-enable this one if you think these tests are important to keep.

Reply:
Either way is OK.
We can skip this test for now.
This feature is not actually used in any logic files as long as I know.